testsuite/integration/microprofile-tck/telemetry/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
License header missing

Reply:
Copied and pasted from another test. It doesn't seem we put licenses on these test files.